spec/system/hiring_staff_have_to_accept_terms_spec.rb

```diff
@@ -66,6 +67,10 @@
   end
 
   context "the user has accepted the terms and conditions" do
+    before do
+      stub_publishers_auth(urn: school.urn, oid: oid)
```

Comment:
Now we're using Devise, do we need to have this helper method at all? Could we just use devise `login_as`?

Reply:
we still need to associate a publisher with an organisation, that is the next refactoring that I am doing

Reply:
Yeah I just realised that a second ago on my branch 😂